conda/core/index.py

```diff
@@ -157,6 +158,11 @@ def _supplement_index_with_system(index):
         rec = _make_virtual_package('__cuda', cuda_version)
         index[rec] = rec
 
+    libc_family, libc_version = context.libc_family_version
```

Comment:
I'm wondering a bit on how this will behave on systems where we have musl + glibc-wrapper, e.g. the `miniconda3:4.7.10-alpine` docker image.

Reply:
Yes, that image gives you glibc, 2.28

Reply:
Can you point me to where the tests actually fail? All of the Circle tests I saw seem to fail long before this

Reply:
I was referring to https://ci.appveyor.com/project/ContinuumAnalyticsFOSS/conda/builds/28297712/job/e0in8bb058fta1ep#L904